libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Util/SFManagedPreferences.m

```diff
@@ -84,55 +86,56 @@ - (id)init {
         if (self.rawPreferences) {
             [[SalesforceSDKManager sharedManager] registerAppFeature:kSFAppFeatureManagedByMDM];
         }
-        NSMutableDictionary *attributes = [[NSMutableDictionary alloc] init];
-        if (self.rawPreferences) {
-            attributes[@"mdmIsActive"] = [NSNumber numberWithBool:YES];
-            attributes[@"mdmConfigs"] = self.rawPreferences;
-        } else {
-            attributes[@"mdmIsActive"] = [NSNumber numberWithBool:NO];
-        }
-        [SFSDKEventBuilderHelper createAndStoreEvent:@"mdmConfiguration" userAccount:nil className:NSStringFromClass([self class]) attributes:attributes];
```

Comment:
it has to moved out of SFManagedPreferences init (sharedIntance actually) since that init is called from inside the SFUserAccountManger sharedIntance init call. And internally its trying to init SFUserAccountManger from there.